ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
> So, I'm not sure what to do here (except wrt. the docs, which have to be fixed).

@cmb69 AFAICT the docs do not _precisely_ specify the behavior of the `$regs` parameter when the match fails, however:

> Parameters
>
> pattern The search pattern.
> string  The search string.
> regs    **Contains a substring of the matched string.**
>
> Return Values
>
> Executes the regular expression match with multibyte support, and returns 1 if matches are found. **If the optional regs parameter was specified, the function returns the byte length of matched part, and the array regs will contain the substring of matched string**. The function returns 1 if it matches with the empty string. If no matches are found or an error happens, FALSE will be returned.

To me this sounds like `$regs` should always be an array.